File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_SOlution_Four/C/Program0.c
#include <stdio.h>
#include <math.h>

void main() {

	double l = 0,sum = 0;
	printf("Enter The Length of the Series\n");
	scanf("%lf",&l);
	

	for(double i = 1; i <= l; i++)
		sum+=pow(i,i*(i-1));
	
	printf("Sum of Series = %.0lf\n",sum);
}

File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_SOlution_Four/Java/Program0.java
import java.io.*;

class Program0 {


	public static void main(String[] args) throws IOException {

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		double sum = 0;
		int l = 0;
		System.out.printf("Enter The Length of the Series\n");

		l = Integer.parseInt(br.readLine());


		for(int i = 1; i <= l; i++)
			sum+=Math.pow(i,i*(i-1));

		System.out.println("Sum of Series = "+sum);
	}
}
